contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -83,25 +82,6 @@ def get_package_manager(self, package_manager=None):
       ))
     return package_manager_obj
 
-  @memoized_method
-  def version(self, context=None):
-    # The versions reported by node and embedded in distribution package names are 'vX.Y.Z'.
-    # TODO: After the deprecation cycle is over we'll expect the values of the version option
-    # to already include the 'v' prefix, so there will be no need to normalize, and we can
-    # delete this entire method override.
-    version = super(NodeDistribution, self).version(context)
-    deprecated_conditional(
-      lambda: not version.startswith('v'), entity_description='', removal_version='1.7.0.dev0',
-      hint_message='value of --version in scope {} must be of the form '
-                   'vX.Y.Z'.format(self.options_scope))
-    return version if version.startswith('v') else 'v' + version
-
-  @classmethod
-  def _normalize_version(cls, version):
```

Comment:
This was removed from the `__init__` [here](https://github.com/pantsbuild/pants/commit/84422c3056edf3ef78d40a429d5316b10215132e), AFAICT it's not used elsewhere either so opted to remove it as well.